### DISCUSSION TITLE
Выбор DI контейнера вместо Unity

### DISCUSSION THREAD

**Discussion by kuzkok** [Flexberry/rfcs#16] [General]
Unity вроде как приказал долго жить, надо что-то выбрать вместо него - вариант DI контейнер из коробки vs [autofac](https://autofac.readthedocs.io/en/latest/index.html). В любом случае потребуется убрать всё упоминание unity из технологических классов.

**Comment by kuzkok** [comment]
Я считаю что чем проще тем лучше - в технологии по умолчанию генерить использование стандартного контейнера, те проекты которым требуется что-то этакое смогут сами подключить для себя autofac.

**Reply by mao29** [reply]
Согласен, нужно написать задачу на выпиливание повсюду в технологии UnityFactory, а также переписать [код регистрации сервисов в шаблоне генерируемого приложения](https://github.com/Flexberry/CodeGenSample.AspNetCorePlusEmber/blob/5eaefa326dbf9524a44f102ac33e162c21a84fa1/src/SuperSimpleContactList/ODataBackend/Startup.cs#L116) и желательно сделать пример как подружить технологию с autofac-ом.

**Comment by bratchikov** [comment]
Предусмотреть возможность оставить на проекте Unity после обновления до ORM 7.
